src/libstore/ssh-store.cc

```diff
@@ -82,7 +84,9 @@ ref<FSAccessor> SSHStore::getFSAccessor()
 ref<RemoteStore::Connection> SSHStore::openConnection()
 {
     auto conn = make_ref<Connection>();
-    conn->sshConn = master.startCommand("nix-daemon --stdio");
+    conn->sshConn = master.startCommand(
```

Comment:
Might not be worth the effort, but this could be generalised between the 2 implementations to avoid some of duplication.